testing/jupyterlab/playwright.config.ts

```diff
@@ -71,7 +71,7 @@ const config: PlaywrightTestConfig = {
 
   /* Run a server. The tests will open urls to this server in the browser. */
   webServer: {
-    command: "yarn start",
+    command: "npm start",
```

Comment:
This is the command that's now causing the GitHub action to fail. It's defined in package.json:

```
    "start": "jupyter lab --config ./jupyter_server_config.py"
```

Based on the logs, it seems like the `jupyter` command here is failing, but why?